service/app/controllers/bulkapimanagement/validator/BulkUploadRequestValidator.java

```diff
@@ -0,0 +1,24 @@
+package controllers.bulkapimanagement.validator;
+
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.BaseRequestValidator;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+
+public class BulkUploadRequestValidator extends BaseRequestValidator {
+
+  public void validateEncryptRequest(Request request) {
+    commonValidation(request);
+  }
+
+  public void validateDecryptRequest(Request request) {
+    commonValidation(request);
+  }
+
+  private void commonValidation(Request request) {
+    if (request.getRequest().get(JsonKey.USER_IDs) == null) {
```

Comment:
if condition is not required - and this will not check blank data case as well. directly pass UserIds in first argument of validateParam